projects/serializers.py

```diff
@@ -85,10 +84,39 @@ class ProjectControlSerializer(serializers.ModelSerializer):
     project = BasicViewProjectSerializer(read_only=True)
     catalog_data = serializers.SerializerMethodField(read_only=True)
     component_data = serializers.SerializerMethodField(read_only=True)
+    disable_narratives = serializers.ListSerializer(child=serializers.IntegerField(), write_only=True, required=False)
+    enable_narratives = serializers.ListSerializer(child=serializers.IntegerField(), write_only=True, required=False)
 
     class Meta:
         model = ProjectControl
-        fields = ("status", "project", "control", "catalog_data", "component_data", )
+        fields = (
+            "status", "project", "control", "catalog_data", "component_data", "disable_narratives", "enable_narratives"
+        )
+
+    def update(self, instance, validated_data):
+        if disable := validated_data.get("disable_narratives"):
+            instance.disabled_narratives = list(set(instance.disabled_narratives).union(disable))
+
+        if enable := validated_data.get("enable_narratives"):
+            instance.disabled_narratives = list(set(instance.disabled_narratives).difference(enable))
```

Comment:
Set union to avoid adding duplicate values. Need to validate the input data so that only allowed values can be added (and to avoid this list growing arbitrarily large since it's unconstrained against the list of project.components.

Set difference removes only those items actually contained in the original set and disregards extras.